include/seastar/core/simple-stream.hh

```diff
@@ -148,6 +149,7 @@ public:
     void skip(size_t size) {
         for_each_fragment(size, [] (auto) { });
     }
+
```

Comment:
nit: please refrain from mixing cosmetic changes with functional changes.
In this case, the cosmetic changes don't seem to be required at all...